cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,26 @@
-describe('', () => {
-  before(() => {
+/* eslint-disable */
+/// <reference types='cypress' />
+describe('check article flow', () => {
+  beforeEach(() => {
+    cy.task('generateUser').then((userDataString) => {
+      const user = new Object(userDataString)
```

Comment:
The use of `new Object(userDataString)` is incorrect. If `userDataString` is a JSON string, it should be parsed using `JSON.parse(userDataString)` to convert it into a JavaScript object.